initdb.py

```diff
@@ -3,7 +3,7 @@
 import os
 import csv
 import mysql.connector
-
+from exceptions import *
```

Comment:
Don't use wildcard imports like this. See [here](http://stackoverflow.com/a/193931/4167059) for a couple examples.